packages/element/src/react.js

```diff
@@ -133,3 +133,24 @@ export function switchChildrenNodeName( children, nodeName ) {
 		return createElement( nodeName, { key: index, ...props }, childrenProp );
 	} );
 }
+
+/**
+ * Function that returns an array of all the text elements
```

Comment:
I don't understand this comment. I think "descendants" should be "descendant". Could you rephrase this maybe?